src/main/jbake/content/hawkular-services/docs/quickstart-guide/index.adoc

```diff
@@ -55,20 +55,273 @@ endif::[]
+    "end": 1468548000000,
+    "min": 9.5,
+    "avg": 9.55,
+    "median": 9.5,
+    "max": 9.6,
+    "sum": 38.2,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468548000000,
+    "end": 1468555200000,
+    "min": 9.5,
+    "avg": 9.5,
+    "median": 9.5,
+    "max": 9.5,
+    "sum": 38,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468555200000,
+    "end": 1468562400000,
+    "min": 9.6,
+    "avg": 9.649999999999999,
+    "median": 9.6,
+    "max": 9.7,
+    "sum": 38.6,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468562400000,
+    "end": 1468569600000,
+    "min": 9.8,
+    "avg": 10.075,
+    "median": 10,
+    "max": 10.3,
+    "sum": 40.3,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468569600000,
+    "end": 1468576800000,
+    "min": 9.7,
+    "avg": 9.924999999999999,
+    "median": 9.9,
+    "max": 10.1,
+    "sum": 39.699999999999996,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468576800000,
+    "end": 1468584000000,
+    "min": 9.2,
+    "avg": 9.825,
+    "median": 9.2,
+    "max": 11,
+    "sum": 39.3,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468584000000,
+    "end": 1468591200000,
+    "min": 12.4,
+    "avg": 13.399999999999999,
+    "median": 13.3,
+    "max": 14.3,
+    "sum": 53.6,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468591200000,
+    "end": 1468598400000,
+    "min": 10.1,
+    "avg": 10.399999999999999,
+    "median": 10.2,
+    "max": 11,
+    "sum": 41.599999999999994,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468598400000,
+    "end": 1468605600000,
+    "min": 10.9,
+    "avg": 11.375,
+    "median": 11.4,
+    "max": 11.7,
+    "sum": 45.5,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468605600000,
+    "end": 1468612800000,
+    "min": 9.6,
+    "avg": 10.35,
+    "median": 10.1,
+    "max": 11.1,
+    "sum": 41.4,
+    "samples": 4,
+    "empty": false
+  },
+  {
+    "start": 1468612800000,
+    "end": 1468620000000,
+    "min": 9.1,
+    "avg": 9.125,
+    "median": 9.1,
+    "max": 9.2,
+    "sum": 36.5,
+    "samples": 4,
+    "empty": false
+  }
+]
+----
+
+.Learn more
+NOTE: We only scratched the surface of what we can do when storing and retrieving metrics, in the link:/hawkular-metrics/docs/user-guide/#_downsampling[Metrics User Guide]
+you will see how to tag metrics, aggregate values from various metrics and other advanced features.
+
+== Step 3: Add alert definition
+
+We saw how to add metrics, visualize those and make some computation, now we'll see how to be notified for particular conditions.
+An alert trigger consists of:
+
+- A set of conditions to trigger the alert/event/notification
+- One or multiple actions to take when the conditions are met (send an email, call a webhook, start/stop new docker containers or VM for autoscaling...)
+- Some additional metadata (such as the severity, a description...)
```

Comment:
(severity, description or business context)
We can add generic context to extend the alerting model with business specific info.
I guess this could be useful to note it.